### PR TITLE
Get user.ts to 100% coverage

### PR DESCRIPTION
**Pull request by sdevalapurkar**
# Overview

Get user.ts to 100% coverage

image

### CODE REVIEW

File: api/src/paths/user/self.test.ts [hunk 1]
```diff
@@ -64,7 +64,7 @@ describe('getUser', function () {
       const result = self.getUser();
 
       // eslint-disable-next-line
-      //@ts-ignore
```
Comment (NickPhura) [inline api/src/paths/user/self.test.ts]: If we cast the value to an `any` does it solve this issue (or possibly the unknown/any thing)?  Might be nicer than ignores everywhere?

Comment (sdevalapurkar) [reply api/src/paths/user/self.test.ts]: the issue is with it missing like 76 fields that I dont need to mock out

Comment (NickPhura) [reply api/src/paths/user/self.test.ts]: Should be able to do something like:

```Typescript
  const sampleReq = {
    keycloak_token: {}
  } as any;
    
  await result(sampleReq, (null as unknown) as any, (null as unknown) as any);
```

And then won't need any of the ignores

Comment (sdevalapurkar) [reply api/src/paths/user/self.test.ts]: thanks!

**Review by NickPhura — DISMISSED** [review]

**Review by NickPhura — APPROVED** [review]

**Pull request merged**